drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_equity_analysis.haml

```diff
@@ -59,3 +59,7 @@
             No report data has been selected to display.
           %span
             Make required selections above and click “update report”
+  - if @analysis_builder.valid?
```

Comment:
Should this be
```suggestion
  - unless @analysis_builder.valid?
```
?

Reply:
No, that's basically show if we've made some selections.  It's not a warning about poor choices, but a warning about which data is available.